cheroot/workers/threadpool.py

```diff
@@ -167,18 +175,14 @@ def __init__(
         self._pending_shutdowns = collections.deque()
 
     def start(self):
-        """Start the pool of threads."""
-        for _ in range(self.min):
-            self._threads.append(WorkerThread(self.server))
-        for worker in self._threads:
-            worker.name = (
-                'CP Server {worker_name!s}'.
-                format(worker_name=worker.name)
-            )
-            worker.start()
-        for worker in self._threads:
-            while not worker.ready:
-                time.sleep(.1)
+        """Start the pool of threads.
+
+        Raises:
+            RuntimeError: if the pool is already started
+        """
+        if self._threads:
+            raise RuntimeError('threadpools can only be started once')
```

Comment:
I think exception messages usually start with a capital letter.
```suggestion
            raise RuntimeError('Threadpools can only be started once')
```